frontend/uddhar-frontend/src/coordinator/components/OrgDetailOverlay.jsx

```
@@ -1,44 +1,51 @@
+};
 
-  // Move this outside of the DetailOverlay component
-  const DetailRow = ({ label, value, isLink }) => (
-    <div className="border-b py-3">
-      <label className="text-sm text-gray-500 block">{label}</label>
-      <div className="mt-1">
-        {isLink ? renderLink(value, value) : value || "N/A"}
-      </div>
+const DetailRow = ({ label, value, isLink }) => (
+  <div className="border-b py-3">
+    <label className="text-sm text-gray-500 block">{label}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)